build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
I have opened issue #777 for scenario 2. Scenario 3(E2E) is a manual verification, just need to manual change the version to the latest in the E2E app. No automation work needed.